### DISCUSSION TITLE
Is there anything included to handle cleaning up BepInEx/LogOutput.log?

### DISCUSSION THREAD

**Discussion by jamieastley** [mbround18/valheim-docker#599] [Q&A] [answered]
I didn't see anything in the docs regarding this, so as per title: is there any sort of tool/script included within the Docker image to handle cleaning up the BepInEx log file on a regular basis?

I was previously using GPortal for my dedicated server, however am looking to move to hosting this Docker image on AWS, and am hesitant about this log file creeping in size much like it has on my GPortal instance?

![CleanShot 2022-07-18 at 14 07 31](https://user-images.githubusercontent.com/6836556/179444158-26a45f87-4f5d-4cba-b205-8c15f1641258.png)

**Comment by mbround18** [comment, marked as answer]
are you running in debug mode? i typically use log rotate for these

**Reply by jamieastley** [reply]
assuming the config option `WriteUnityLog` is what you're referring to, then yes it's enabled (though I don't recall ever enabling this myself...).

Definitely somewhat user error here as it's likely due to Valheim auto-updating, and this hosted instance is/was running mods so likely endless exception spam getting logged (I'm not about to try downloading it to verify 😅).

Thanks for `logrotate` pointer! Saves me from writing my own, shittier implementation of exactly that.
